sda_uploader/gui.py

```diff
@@ -1,8 +1,11 @@
 """SDA Uploader GUI."""
 
+import os
```

Comment:
if importing os just of chmod then
```suggestion
from os import chmod
```